pkg/ccl/crosscluster/logical/lww_row_processor.go

```diff
@@ -574,10 +575,10 @@ VALUES (%s)
 ON CONFLICT (%s)
 DO UPDATE SET
 %s
-WHERE (t.crdb_internal_mvcc_timestamp <= excluded.crdb_replication_origin_timestamp
-     AND t.crdb_replication_origin_timestamp IS NULL)
- OR (t.crdb_replication_origin_timestamp <= excluded.crdb_replication_origin_timestamp
-     AND t.crdb_replication_origin_timestamp IS NOT NULL)`
+WHERE (t.crdb_internal_mvcc_timestamp <= $%[6]d
+    AND t.crdb_internal_origin_timestamp IS NULL)
+ OR (t.crdb_internal_origin_timestamp <= $%[6]d
```

Comment:
what does the 6 do?  I believe the `$%[6]d` is grabbing the local value's originTimestamp value, but i'm not sure how.

Reply:
ah, i think i got hung up on the fact that we still throw the origin timestamp in the scratch datums buffer. for better readability, we may want the querier to have a separate buffer for the origin timestamp.

Reply:
I agree there is some cleanup that can happen here. But, I'm kinda wondering if this whole query builder interface should just be torn up a bit, but I didn't want to do a larger refactor in this PR.

Reply:
Totally fair. we can do this later.